modules/videoio/src/cap_images.cpp

```diff
@@ -213,7 +213,12 @@ static char* icvExtractPattern(const char *filename, unsigned *offset)
         unsigned int dummy;
         if(sscanf(at + 1, "%ud", &dummy) != 1)
             return 0;
+#ifdef UNDER_RTSS
+        name = _strdup(filename);
+#else
```

Comment:
We really don't that such changes.

You can move such "glue" into some "rtss_port.hpp" header where to re-implement these functions:
```
static inline
char* rtss_stddup(const char *s)
{
    return _strdup(s);
}
#define strdup rtss_stddup
```